core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -130,7 +128,8 @@ public void doWork(PathWrapper altRsp, List<Path> paths, Translation tr) {
             altRsp.setInstructions(fullInstructions);
 
         if (!allFound) {
-            altRsp.addError(new ConnectionNotFoundException("Connection between locations not found", Collections.<String, Object>emptyMap()));
+            altRsp.addError(new ConnectionNotFoundException("Connection between locations not found: " +
```

Comment:
Since we now know which locations/points create this issue, wouldn't it be better to state "Connection between point1 and point2 (, and point 5 and point 6) not found"? IMHO the (1,2) look a bit like coordinates (at least I didn't understand the error message on first sight).